guides/source/active_record_basics.md

```diff
@@ -326,12 +326,11 @@ into the database. There are several methods that you can use to check your
 models and validate that an attribute value is not empty, is unique and not
 already in the database, follows a specific format, and many more.
 
-Validation is a very important issue to consider when persisting to the database, so
-the methods `save` and `update` take it into account when
-running: they return `false` when validation fails and they don't actually
-perform any operations on the database. All of these have a bang counterpart (that
-is, `save!` and `update!`), which are stricter in that
-they raise the exception `ActiveRecord::RecordInvalid` if validation fails.
+Methods like `save`, `create` and `update` validate a model before persisting
+it to the database. When a model is invalid these methods return `false` and no
+database operation gets performed. All of these methods have a bang counterpart
```

Comment:
```suggestion
database operation will be performed. All of these methods have a bang counterpart
```

Reply:
Or maybe "database operations are performed" in the plural because there may be more than one operation.

Reply:
Thanks. Went with "database operations are performed".